src/services/sessions.js

```diff
@@ -2308,6 +2077,22 @@ module.exports = class SessionsHelper {
 				})
 			}
 
+			const defaults = await getDefaults()
+			if (!defaults.orgCode) {
+				return responses.failureResponse({
+					message: 'DEFAULT_ORG_CODE_NOT_SET',
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			}
+			if (!defaults.tenantCode) {
+				return responses.failureResponse({
+					message: 'DEFAULT_TENANT_CODE_NOT_SET',
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			}
+
```

Comment:
Can we remove this since we are not using it in the function?

`const tenantCodes = [tenantCode, defaults.tenantCode]`
`const orgCodes = [orgCode, defaults.orgCode]`

These two variables are not used anywhere.

If we have this logic in any other place, we should remove it, since we should never share data between tenants.